### DISCUSSION TITLE
当使用阅读模式页面出现卡顿时的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1685] [重要通知]
## 问题描述

RT

## 如何排查

可以在 **隐身模式下，仅启用简悦** 看看是否仍存在此问题？（通过此方式，可排除掉其它扩展造成的影响）

## 解决方案

1. 停用简悦的 [目录功能 ](http://ksria.com/simpread/docs/#/%E7%9B%AE%E5%BD%95)
2. 停用 **如何排查** 得到的某个插件

上述两个方案使用其中一种即可解决。

**Comment by 210000ling** [comment]
我的卡顿是出现在少数派的(https://sspai.com/post/*)  页面,.是使用vimum插件滚动页面的时候卡顿,使用鼠标滑动和触控板时不卡顿.停用了目录功能后依然卡顿,把全部插件关闭后依然卡顿.

**Reply by Kenshin** [reply]
这样的话，目前也没有太好的解决方案，同样的操作在其它页面有问题吗？
